### PR TITLE
Release 2018 09

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- merging weekly into source for 2018.09

### Resolves:
 [OKTA-159656](https://oktainc.atlassian.net/browse/OKTA-159656)

### CODE REVIEW

File: _source/_change-log/2018-07.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The following feature enhancement is expected in preview orgs February 14, 2018,
 
 #### Keystore Rollover Events Now Logged
 
-OAuth key store rollover events are now included in both the [Events](/docs/api/resources/events) and [System Log](/docs/api/resources/system_log) APIs.<!-- OKTA-129535 -->
+OAuth key store rollover events are now included in both the [Events](https://developer.okta.com/docs/api/resources/events) and [System Log](https://developer.okta.com/docs/api/resources/system_log) APIs.<!-- OKTA-129535 -->
```
Comment (mystiberry-okta) [inline _source/_change-log/2018-07.md]: These shouldn't be full URLs, will fix.

File: _source/_docs/api/resources/authn.md [hunk 2]
```diff
@@ -3599,8 +3599,13 @@ Parameter      | Description                                         | Param Typ
 -------------- | --------------------------------------------------- | ---------- | -------- | -------- |
 factorId            | `id` of factor returned from enrollment             | URL        | String   | TRUE     |
 stateToken     | [state token](#state-token) for current transaction | Body       | String   | TRUE     |
+<<<<<<< HEAD
+rememberDevice | user&#8217;s decision to remember device            | URL        | Boolean  | FALSE    |
+autoPush       | user&#8217;s decision to send push to device automatically | URL | Boolean  | FALSE    |
+=======
 rememberDevice | user's decision to remember device            | URL        | Boolean  | FALSE    |
 autoPush       | user's decision to send push to device automatically | URL | Boolean  | FALSE    |
+>>>>>>> origin/source
```
Comment (mystiberry-okta) [inline _source/_docs/api/resources/authn.md]: This is an incomplete conflict, will fix.

File: _source/_docs/api/resources/authn.md [hunk 3]
```diff
@@ -3599,8 +3599,8 @@ Parameter      | Description                                         | Param Typ
 -------------- | --------------------------------------------------- | ---------- | -------- | -------- |
 factorId            | `id` of factor returned from enrollment             | URL        | String   | TRUE     |
 stateToken     | [state token](#state-token) for current transaction | Body       | String   | TRUE     |
-rememberDevice | user's decision to remember device            | URL        | Boolean  | FALSE    |
-autoPush       | user's decision to send push to device automatically | URL | Boolean  | FALSE    |
+rememberDevice | user&#8217;s decision to remember device            | URL        | Boolean  | FALSE    |
```
Comment (jmelberg-okta) [inline _source/_docs/api/resources/authn.md]: Please use `'` instead of `&#8217;`

File: _source/_docs/api/resources/sessions.md [hunk 4]
```diff
@@ -564,6 +564,28 @@ If the session is invalid, a `404 Not Found` response will be returned.
...
+
+Example Request
+
+~~~http
+Accept: application/json
+Content-Type: application/json
+Authorization: SSWS ${api_token} or {sid_cookie}
+Prefer: return=minimal
+
+POST https://${yourDomain}.com/api/v1/sessions/{id-or-me}/refresh
```
Comment (jmelberg-okta) [inline _source/_docs/api/resources/sessions.md]: -> `{yourOktaDomain}`

Comment (mystiberry-okta) [reply _source/_docs/api/resources/sessions.md]: Oh man, that is not the latest version. What is up?!

Comment (mystiberry-okta) [reply _source/_docs/api/resources/sessions.md]: I changed it to curl, too.

Comment (mystiberry-okta) [reply _source/_docs/api/resources/sessions.md]: Oh, shoot, I changed in RN, not in source. doh. no mystery.

**Pull request merged**